cms/djangoapps/contentstore/tests/test_import_pure_xblock.py

```diff
@@ -60,8 +62,11 @@ def _assert_import(self, course_dir, expected_field_val, has_draft=False):
                 the expected field value set.
 
         """
+        # It is necessary to use the "old mongo" modulestore because split doesn't work
+        # with the "has_draft" logic below.
+        store = modulestore()._get_modulestore_by_type(ModuleStoreEnum.Type.mongo)  # pylint: disable=protected-access
         courses = import_from_xml(
-            self.store, self.user.id, TEST_DATA_DIR, [course_dir]
+            store, self.user.id, TEST_DATA_DIR, [course_dir], create_course_if_not_present=True
```

Comment:
Since I wasn't able to change this to split, create_course_if_not_present could be removed. But I think it makes sense to keep it.